### PR TITLE
Updated PHPCS: Modules, endpoints, and general files

### PR DESCRIPTION
**Pull request by kraftbj**
After #17406, updated the "requiredlist" files to match new PHPCS rules for packages and the debugger.

#### Changes proposed in this Pull Request:
* PHPCS

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
n/a

#### Testing instructions:
n/a

#### Proposed changelog entry for your changes:
* n/a

### CODE REVIEW

**Comment by kraftbj** [comment]
After 9.1 branches off and #17406 merges, we need to update base branch to `master`, then this can be merged.

**Comment by anomiex** [comment]
> After 9.1 branches off and #17406 merges, we need to update base branch to `master`, then this can be merged.

Once #17406 merges and its branch is deleted, Github will automatically set the base branch to master (or whatever we're calling it by then).

**Review by anomiex — DISMISSED** [review]
Looks sane to me.

Of the files in this patch currently listed by Fusion as not being in sync and where we're removing `use` statements,

* _inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-tweetstorm-gather.php reports no diff.
* modules/search/class-jetpack-instant-search.php shows no uses of those `use`s in the diff.
* modules/shortcodes/instagram.php reports no diff.

**Review by anomiex — CHANGES_REQUESTED** [review]
Needs a manual rebase after #17595. Otherwise should be good.

File: modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php [hunk 1]
```diff
@@ -40,10 +40,9 @@ private function __construct() {
 			add_filter( 'admin_body_class', array( $this, 'add_iframed_body_class' ) );
 		}
 
-		require_once dirname( __FILE__ ) . '/functions.editor-type.php';
+		require_once __DIR__ . '/functions.editor-type.php';
 		add_action( 'edit_form_top', 'Jetpack\EditorType\remember_classic_editor' );
 		add_filter( 'block_editor_settings', 'Jetpack\EditorType\remember_block_editor', 10, 2 );
-
```
Comment (anomiex) [inline modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php:46]: Lost a newline here in the rebase. Probably doesn't matter.

**Review by anomiex — APPROVED** [review]

**Review by sergeymitr — APPROVED** [review]

**Comment by kraftbj** [comment]
r216013-wpcom

**Pull request merged**